### PR TITLE
Feat 18/auth

### PR DESCRIPTION
**Pull request by sueelly**
## #️⃣ Related Issue

Linked Feature backlog: close (put issue number here with #)

## 📝 Purpose of PR

Please briefly explain your work in this PR.

## Contents

List of work items performed.

## Checklist

-   [ ] Does commit messages follow the convention?
-   [ ] Are variable names brief but descriptive?
-   [ ] Is the code clean and easy to understand?
-   [ ] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
-   [ ] Does the test successfully test edge and corner cases?

## 💬 Review Requirements (Optional)

Is there anything you can praise about this PR? Start with praise.
Make a review, leaving kind comments and suggesting changes where needed (to resolve the above).

### CODE REVIEW

File: snapnow-api/src/main/java/kr/flab/snapnow/auth/service/AuthService.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import kr.flab.snapnow.auth.model.dto.response.TokenResponse;
+import kr.flab.snapnow.auth.model.dto.request.EmailLoginRequest;
+import kr.flab.snapnow.auth.model.dto.request.TokenReissueRequest;
+import kr.flab.snapnow.auth.model.dto.request.SignUpRequest;
+
+@Service
+@RequiredArgsConstructor
+public class AuthService {
+
+    private final JwtProvider jwtProvider;
```
Comment (f-lab-jason) [inline snapnow-api/src/main/java/kr/flab/snapnow/auth/service/AuthService.java:19]: 이 친구는 어디에 있는건가요..?

**Review by f-lab-jason — COMMENTED** [review]
으음..이 pr보다 뒤에 있는 친구들은 정리를 좀 하면 좋을 것 같아요..
뭐부터 어딜 어떻게 봐야할지 잘..모르겠네요..ㅜㅜㅜ

File: snapnow-api/src/main/java/kr/flab/snapnow/auth/service/AuthService.java (hunk 1 above)
Comment (sueelly) [reply snapnow-api/src/main/java/kr/flab/snapnow/auth/service/AuthService.java:19]: 바로 위 PR에 있는데, auth에서 필요해서 우선 넣어서 올렸습니다!

Comment (sueelly) [reply snapnow-api/src/main/java/kr/flab/snapnow/auth/service/AuthService.java:19]: @f-lab-jason 전체 갈아엎어서 위에 domain init부터 보시면 될 것 같습니다...!
모델의 경우에는 거의 비슷한데 조금 더 분리가 되었다고 보시면 될 것 같습니다
